pkg/compilers/names.go

```diff
@@ -45,6 +45,7 @@ var (
 	RUMP_JAVA_QEMU       = compilerName("rump", "java", "qemu")
 
 	OSV_JAVA_XEN        = compilerName("osv", "java", "xen")
+	OSV_JAVA_OPENSTACK  = compilerName("osv", "java", "openstack")
```

Comment:
Not that it matters, but is there a reason for inserting it here? Wouldn't it make more sense to append it at the end of JAVA list?